crates/holochain/src/core/ribosome/host_fn/get_links.rs

```diff
@@ -84,6 +84,16 @@ pub mod slow_tests {
             TestString::from("foo.bar".to_string())
         );
 
+        // ensure a really long path
+        println!("x");
+        let _: () = crate::call_test_ribosome!(
+            host_access,
```

Comment:
This will fail sys validation but `call_test_ribosome` doesn't call the call zome workflow so your not running the validation. If you were to `call_zome` on the handle this would fail.